_pages/en/capabilities.md

```diff
@@ -471,31 +464,25 @@ InfoSec supplies development teams and operational teams with an ample number of
 
 #### 3.2.2. Indicators  
 
-- How often your team monitors the stability and security of your IT system?  
-   a. Very Frequently  
-   b. Frequently  
-   c. Occasionally  
-   d. Rarely  
-   e. Very Rarely  
-   f. Never  
+Ideal: Very Frequently  
 
-- How do you notify your team about failure alerts?  
-   a. Via email  
-   b. Via PR in Github  
-   c. Via Slack  
-   d. Via Trello board  
-   e. Via phone call  
-   f. None of the above/not applicable  
+- How often your team monitors the stability and security of your IT system?  
```

Comment:
What's the source for this question? It seems to be up to interpretation. 

If I monitor the stability of my apps (automatically) does that count as "very frequently", or if I automate the monitoring of my tool but only check if it there is an issue, then would that be "Rarely" even though they're following best practice?